tests/vmi_configuration_test.go

```diff
@@ -2508,6 +2508,20 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				pinnedCPUsList, err := hw_utils.ParseCPUSetLine(output, 100)
 				Expect(err).ToNot(HaveOccurred())
 
+				output, err = tests.ListCgroupThreads(readyPod)
+				Expect(err).ToNot(HaveOccurred())
+				pids := strings.Split(output, "\n")
```

Comment:
Probably also need to ensure that `len(pids) > 0`

Reply:
For the case with QEMU TCG, all threads will have been moved to the housekeeping cgroup.
Therefore to avoid QEMU TCG test failing, should accept the case where len(pids)==0.

Reply:
In such case, what do you think about having a separate test case for TCG? Because now if it's not TCG, the test may hide the actual problem.

Reply:
I don't think its necessary: vCPU isolation should not be used along with TCG. So don't see a point in testing it ?

Reply:
A test should have a deterministic behavior. If you are testing vCPU isolation, then IMHO you need to specify the exact number of threads that are expected to be in that cgroup and ensure the expectation is fulfilled.

> I don't think its necessary: vCPU isolation should not be used along with TCG. So don't see a point in testing it ?

Sounds like a valid testcase: ensure that vCPU isolation is *not* ised with TCG.

Reply:
The test does have a deterministic behaviour: If any thread exists on the main pod cgroup, with a name
which does not contain "QEMU VCPU", then the test fails.